modules/sdk-core/src/bitgo/utils/tss/ecdsa/ecdsa.ts

```diff
@@ -481,6 +481,16 @@ export class EcdsaUtils extends baseTSSUtils<KeyShare> {
     if (!bitGoToRecipientShare) {
       throw new Error('Missing BitGo to User key share');
     }
+    // check the wallet signature for this share
+    const bitGoToRecipientPrivateShare = await this.decryptPrivateShare(bitGoToRecipientShare.privateShare, userGpgKey);
+
+    await this.verifyWalletSignatures(
+      userGpgKey.publicKey,
+      userGpgKey.publicKey,
```

Comment:
will get changed to `backupGpgKey` once https://github.com/BitGo/BitGoJS/pull/3184 is merged in